peachjam/templates/peachjam/gazette_year.html

```diff
@@ -12,16 +12,25 @@
             <li class="breadcrumb-item">
               <a href="{% url 'gazettes' %}">{% trans 'Gazettes' %}</a>
             </li>
+            {% if country %}
+              <li class="breadcrumb-item">
+                <a href="{% url 'gazettes_by_locality' country.pk.lower %}">{{ country }}</a>
+              </li>
+            {% endif %}
             {% if locality %}
               <li class="breadcrumb-item">
-                <a href="{% url 'gazettes_by_locality' code=locality.code %}">{{ locality }}</a>
+                <a href="{% url 'gazettes_by_locality' locality.place_code %}">{{ locality }}</a>
               </li>
             {% endif %}
+            <li class="breadcrumb-item active">{{ year }}</li>
```

Comment:
in the years list, the  "All years" link isn't working properly, I think it should redirect to the main list page. The is a context variable "all_years_url" thats missing

Reply:
fixed, thanks